core/templates/dev/head/pages/exploration_player/FeedbackPopupDirective.js

```diff
@@ -27,11 +27,13 @@
 // The state-name argument is optional. If it is not provided, the feedback is
 // assumed to apply to the exploration as a whole.
 oppia.directive('feedbackPopup', [
-  'oppiaPlayerService', function(oppiaPlayerService) {
+  'oppiaPlayerService', 'UrlInterpolationService',
+  function(oppiaPlayerService, UrlInterpolationService) {
     return {
       restrict: 'E',
       scope: {},
-      templateUrl: 'components/feedback',
+      templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
+        '/pages/exploration_player/feedback_component_model_directive.html'),
```

Comment:
Should be feedback_popup_directive.html, no? Or perhaps some renaming in general may be needed so that the directive HTML and JS file names match.

Reply:
feedback_popup_directive.html is already present. There were two ng-template divs there. So, that file is split in two. The other one is a template just to use this directive.

Reply:
Perhaps that file should be renamed instead, e.g. to feedback_popup_container_directive.html. We want this directive to have matching HTML and JS file names, just like all the others.

Reply:
done.